gorm/converter.go

```diff
@@ -340,15 +345,27 @@ func (converter *DefaultFilteringConditionConverter) StringArrayConditionToGorm(
 	placeholder := ""
 	for _, str := range c.Values {
 		placeholder += "?, "
-		if val, err := converter.Processor.ProcessStringCondition(ctx, c.FieldPath, str); err == nil {
-			values = append(values, val)
+		if value, err = converter.Processor.ProcessStringCondition(ctx, c.FieldPath, str); err == nil {
+			values = append(values, value)
 			continue
 		}
 
 		values = append(values, str)
 	}
 
-	return fmt.Sprintf("(%s %s %s (%s))", dbName, neg, o, strings.TrimSuffix(placeholder, ", ")), values, assocToJoin, nil
+	switch c.Type {
+	case query.StringArrayCondition_IN:
+		o = "IN"
+
+		return fmt.Sprintf("(%s %s %s (%s))", dbName, neg, o, strings.TrimSuffix(placeholder, ", ")), values, assocToJoin, nil
+
+	case query.StringArrayCondition_ANYOF:
+		o = "&&"
```

Comment:
Maybe this should be "overlaps", "anyof" sounds more like a polymorphic comparison.